v5.10.2 (#34034)
mui/material-ui: CHANGELOG.md

# [Versions](https://mui.com/versions/)

## 5.10.2

<!-- generated comparing v5.10.1..master -->

_Aug 22, 2022_

A big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:

> - ✨ [icons] Sync the Material Icons which changes some [icons' appearance](https://app.argos-ci.com/mui/material-ui/builds/4428) (#33988) @michaldudak

Reviewer: We usually use past tense in this section.

> - ✨ [icons] Sync the Material Icons (#33988) @michaldudak
> - Changes in icons' appearances can be seen [here](https://app.argos-ci.com/mui/material-ui/builds/4428)

Reviewer 2: ```suggestion
- ✨ [icons] Sync the Material Icons which changes some [icons' appearance](https://app.argos-ci.com/mui/material-ui/builds/4428) (#33988) @michaldudak
```

# [Versions](https://mui.com/versions/)

## 5.10.2

<!-- generated comparing v5.10.1..master -->

_Aug 22, 2022_

A big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:

- ✨ @michaldudak synced the Material Icons set with the latest from Google (#33988).  

>   A couple of icons changed their appearance. See the difference [on this Argos build](https://app.argos-ci.com/mui/material-ui/builds/4428]).

Reviewer 3: I have fixed https://github.com/mui/material-ui/releases/tag/v5.10.2, the link was not working.

```suggestion
  A couple of icons changed their appearance. See the difference [on this Argos build](https://app.argos-ci.com/mui/material-ui/builds/4428).
```

_Aug 22, 2022_

A big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:

- ✨ [icons] Sync the Material Icons (#33988) @michaldudak 

> - Changes in icons' appearances can be seen [here](https://app.argos-ci.com/mui/material-ui/builds/4428])

Reviewer: ```suggestion
- ✨ @michaldudak synced the Material Icons set with the latest from Google (#33988).  
  A couple of icons changed their appearance. See the difference [on this Argos build](https://app.argos-ci.com/mui/material-ui/builds/4428]).
```

Reviewer 2: @hbjORbj looks like this is better than my version. Your call!
